[2024-05-21T16:42:28.946+0530] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-05-21T16:42:28.962+0530] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: etl_dag.email_notify manual__2024-05-21T09:12:34.180878+00:00 [queued]>
[2024-05-21T16:42:28.966+0530] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: etl_dag.email_notify manual__2024-05-21T09:12:34.180878+00:00 [queued]>
[2024-05-21T16:42:28.966+0530] {taskinstance.py:2306} INFO - Starting attempt 6 of 7
[2024-05-21T16:42:29.025+0530] {taskinstance.py:2330} INFO - Executing <Task(EmailOperator): email_notify> on 2024-05-21 09:12:34.180878+00:00
[2024-05-21T16:42:29.029+0530] {standard_task_runner.py:63} INFO - Started process 95777 to run task
[2024-05-21T16:42:29.030+0530] {standard_task_runner.py:90} INFO - Running: ['airflow', 'tasks', 'run', 'etl_dag', 'email_notify', 'manual__2024-05-21T09:12:34.180878+00:00', '--job-id', '57', '--raw', '--subdir', 'DAGS_FOLDER/etl_dag.py', '--cfg-path', '/tmp/tmpc5lakjbw']
[2024-05-21T16:42:29.031+0530] {standard_task_runner.py:91} INFO - Job 57: Subtask email_notify
[2024-05-21T16:42:29.107+0530] {task_command.py:426} INFO - Running <TaskInstance: etl_dag.email_notify manual__2024-05-21T09:12:34.180878+00:00 [running]> on host prince-5192
[2024-05-21T16:42:29.203+0530] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_EMAIL='prince.yadav@tothenew.com' AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='etl_dag' AIRFLOW_CTX_TASK_ID='email_notify' AIRFLOW_CTX_EXECUTION_DATE='2024-05-21T09:12:34.180878+00:00' AIRFLOW_CTX_TRY_NUMBER='6' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-05-21T09:12:34.180878+00:00'
[2024-05-21T16:42:29.204+0530] {taskinstance.py:430} INFO - ::endgroup::
[2024-05-21T16:42:29.217+0530] {logging_mixin.py:188} WARNING - /home/prince/airflow-workspace/env/lib/python3.10/site-packages/airflow/utils/email.py:154 RemovedInAirflow3Warning: Fetching SMTP credentials from configuration variables will be deprecated in a future release. Please set credentials using a connection instead.
[2024-05-21T16:42:29.217+0530] {email.py:271} INFO - Email alerting: attempt 1
[2024-05-21T16:42:36.288+0530] {email.py:282} INFO - Sent an alert email to ['prince.yadav@tothenew.com']
[2024-05-21T16:42:38.412+0530] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-05-21T16:42:38.415+0530] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=etl_dag, task_id=email_notify, run_id=manual__2024-05-21T09:12:34.180878+00:00, execution_date=20240521T091234, start_date=20240521T111228, end_date=20240521T111238
[2024-05-21T16:42:38.512+0530] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-05-21T16:42:38.517+0530] {taskinstance.py:3498} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-05-21T16:42:38.517+0530] {local_task_job_runner.py:222} INFO - ::endgroup::
